src/main/resources/com/puppycrawl/tools/checkstyle/checks/design/messages_zh.properties

```diff
@@ -1,10 +1,9 @@
-arrangement.members.before.inner=初始化块，构造函数，字段和方法应位于内部类型之前。
-design.forExtension=類 ''{0}'' 看起來像是為擴展設計的（可以是子類），但方法 ''{1}'' 沒有javadoc，解釋瞭如何安全地執行。\
-  如果類不是為擴展而設計的，請考慮創建類 ''{0}'' final或使方法 ''{1}'' static/final/abstract/empty，或為方法添加允許的
+arrangement.members.before.inner=内部方法/内部接口的定义应位于初始化代码块、构造方法、成员变量和方法的定义之后。
+design.forExtension=既然类 ''{0}'' 设计为可继承的，应为可重写的非空方法 ''{1}'' 添加 javadoc 文档说明。
 final.class=类 {0} 应定义为 final 的。
-hide.utility.class=工具类应隐藏 public 构造器。
-interface.type=接口应描述一种类型，从而必须拥有方法。
+hide.utility.class=工具类不应有 public 或 default 构造器。
+interface.type=接口中不能只有常量没有方法。
 mutable.exception=字段 ''{0}'' 应定义为 final 的。
 one.top.level.class=顶级类 {0} 应位于它自己的源文件中。
 throws.count=Throws 总数： {0,number,integer} 个（最多： {1,number,integer} 个）。
-variable.notPrivate=变量 ''{0}'' 应定义为 private 的，并配置访问方法。
+variable.notPrivate=变量 ''{0}'' 应定义为 private 的，并为其创建访问方法。
```

Comment:
The variable ''{0}'' should be defined as private and create an access method for it.